source/includes/code-examples/delete-many/DeleteManyAsync.cs

```diff
@@ -0,0 +1,58 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class DeleteManyAsync
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        var docs = _restaurantsCollection.Find(Builders<Restaurant>.Filter
```

Comment:
do we really need regex condition for simple deleteMany operation?

Reply:
Would a text search ($text) achieve a similar result more efficiently? If not, I think I'll need to rework the whole example.